oryx-app-mllib/src/main/java/com/cloudera/oryx/app/mllib/kmeans/KMeansUpdate.java

```diff
@@ -140,8 +142,24 @@ public double evaluate(JavaSparkContext sparkContext,
                          PMML model,
                          Path modelParentPath,
                          JavaRDD<String> testData) {
+    KMeansPMMLUtils.validatePMMLVsSchema(model, inputSchema);
     JavaRDD<Vector> testingData = parsedToVectorRDD(testData.map(MLFunctions.PARSE_FN));
-    double cost = pmmlToKMeansModel(model).computeCost(testingData.rdd());
+    List<ClusterInfo> clusterInfoList = KMeansPMMLUtils.read(model);
+
+    double cost;
+    switch (evaluationStrategy) {
+      case "DAVIES-BOULDIN" :
+        cost = Evaluation.daviesBouldinIndex(clusterInfoList, testingData);
```

Comment:
This one's better when it's lower, no? we'd need something like 1 / index to make it larger when better. I am not sure what the range of this index is.